tests/Moq.Tests/ItIsAnyTypeFixture.cs

```diff
@@ -73,7 +73,7 @@ public void Setup_with_It_IsAnyType_and_Returns()
 		{
 			var mock = new Mock<IY>();
 			mock.Setup(m => m.Method<It.IsAnyType>((It.IsAnyType)It.IsAny<object>()))
-			    .Returns(new Func<object, object>(arg => arg));
+				.Returns(new Func<object, object>(arg => arg));
```

Comment:
As above, please revert this and all other whitespace-only changes in this file.